libs/datamodel/connectors/datamodel-connector/src/error.rs

```diff
@@ -79,4 +86,10 @@ pub enum ErrorKind {
     "Native types can only be used if the corresponding feature flag is enabled. Please add this field in your datasource block: `previewFeatures = [\"nativeTypes\"]`"
     )]
     NativeFlagsPreviewFeatureDisabled,
+
+    #[error("Native type {} can not be unique in {}.", native_type, connector_name)]
+    IncompatibleAttributeWithNativeType {
```

Comment:
The enum variant sounds generic while the error message is specifically about unique.

Reply:
ah yes I was going for a more generic error message before, I'll change that